pkg/dag/dag_normal_boot.go

```diff
@@ -63,12 +127,12 @@ func RegisterNormalBoot(s *state.State, g *herd.Graph) error {
 	// Write fstab file
 	s.LogIfError(s.WriteFstabDagStep(g,
 		herd.WithDeps(cnst.OpMountRoot, cnst.OpDiscoverState, cnst.OpLoadConfig),
-		herd.WithWeakDeps(cnst.OpMountOEM, cnst.OpCustomMounts, cnst.OpMountBind, cnst.OpOverlayMount)), "write fstab")
+		herd.WithWeakDeps(cnst.OpKcryptUnlock, cnst.OpMountOEM, cnst.OpCustomMounts, cnst.OpMountBind, cnst.OpOverlayMount)), "write fstab")
```

Comment:
Are these unnecessary? Are just lines 102-103 above enough?

Reply:
no, this is ok so it depends on both, especially now that we cant guarantee the order of them we should depend on both so we know this step is done after both of them have finished.